pylint_odoo/examples/.jslintrc

```diff
@@ -160,7 +160,7 @@
     "constructor-super": "error",
     "curly": "error",
     "default-case": "off",
-    "dot-location": "error",
```

Comment:
Can we switch to the [`property` style](https://eslint.org/docs/rules/dot-location#property) instead of completely disabling? IMO the lint is valid, just the default option is wrong.